src/components/HelloWorld.vue

```diff
@@ -1,17 +1,21 @@
 <script setup lang="ts">
-import { ref } from 'vue';
+import { ref, inject } from 'vue';
 import { Button as WikitButton } from '@wmde/wikit-vue-components';
+import Messages, { MessagesKey } from '@/plugins/MessagesPlugin/Messages';
 
 defineProps<{ msg: string }>();
 
 const count = ref( 0 );
+const $messages = inject( MessagesKey, new Messages() );
```

Comment:
I like that we have adopted the composition API in full. This is a great use case example of provide and inject.